trpcultivate_phenoshare/src/Plugin/TripalImporter/TripalCultivatePhenoshareImporter.php

```diff
@@ -233,6 +286,10 @@ public function stage1(&$form, $form_state, $stage_status = '') {
     $form['file'] = [];
 
     // Other relevant fields here.
+    
+    // This importer does not support using file sources from existing field and local/external source.
+    $form[ $fld_wrapper ]['file']['file_upload_existing'] = ['#access' => 'FALSE'];
+    $form[ $fld_wrapper ]['file']['file_local'] = ['#type' => 'hidden'];
```

Comment:
These should be set in the annotation of the importer.

Reply:
Successfully disabled local source 👍  but the existing field persists and it looks like this field will be available whenever there is a file upload field: https://github.com/tripal/tripal/blob/a6c75eee983ff9261076c5d5e41c50c83a3dd2e6/tripal/src/Form/TripalImporterForm.php#L67

Reply:
Why do you feel so strongly that this needs to be removed?

I would like a centralized file source in keeping with the success of the current version, the only field that takes file to be the html5 file field, which is amazing and intuitive as is and I enjoyed clicking using each time. I feel the existing file field is inessential and in some sense perplexing as it is suggesting file that already have been uploaded.

Reply:
Ok, good enough reasoning for me :-)